doc/lua_api.txt

```diff
@@ -3848,6 +3848,12 @@ Environment access
     * Returns `ObjectRef`, or `nil` if failed
 * `minetest.add_item(pos, item)`: Spawn item
     * Returns `ObjectRef`, or `nil` if failed
+* `minetest.item_pickup(item_luaentity, [inventory, picker, ...])`: Pick up a spawned item.
```

Comment:
I don't think this is supposed to be documented, as none of its peers (`core.item_drop`, `core.place_node`, `core.dig_node`, etc.) are.

Reply:
> none of its peers (`core.item_drop`, `core.place_node`, `core.dig_node`, etc.) are.

You're wrong, I think:
https://github.com/minetest/minetest/blob/60bee4d5cdefc5137c330dd317c6e51d3b1cde0f/doc/lua_api.txt#L4290-L4291
https://github.com/minetest/minetest/blob/60bee4d5cdefc5137c330dd317c6e51d3b1cde0f/doc/lua_api.txt#L3827-L3828
https://github.com/minetest/minetest/blob/60bee4d5cdefc5137c330dd317c6e51d3b1cde0f/doc/lua_api.txt#L3829-L3830